projects/ViTDet/README.md

```diff
@@ -0,0 +1,120 @@
+# ViTDet
+
+## Description
+
+<!-- Share any information you would like others to know. For example:
+Author: @xxx.
+This is an implementation of \[XXX\]. -->
+
+This is an implementation of [ViTDet](https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet) based on [MMDetection](https://github.com/open-mmlab/mmdetection/tree/3.x), [MMClassification](https://github.com/open-mmlab/mmclassification/tree/1.x), [MMCV](https://github.com/open-mmlab/mmcv), and [MMEngine](https://github.com/open-mmlab/mmengine).
+
+## Usage
+
+<!-- For a typical model, this section should contain the commands for training and testing. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
+
+Please install mmcls>=1.0, because we need to use `LN2d`.
+
+### Training commands
+
+In MMDetection's root directory, run the following command to train the model:
+
+```bash
+python tools/train.py projects/ViTDet/configs/mask-rcnn_vit-base_fpn_rpn-2conv_4conv1fc_ln-all_lsj-100e_coco.py
+```
+
+Below is an example of using 32 GPUs to train Mask R-CNN on a Slurm partition named _dev_, and set the work-dir to some shared file systems.
+
+```shell
+GPUS=32 ./tools/slurm_train.sh dev vitdet projects/ViTDet/configs/mask-rcnn_vit-base_fpn_rpn-2conv_4conv1fc_ln-all_lsj-100e_coco.py /nfs/xxxx/mask_rcnn_r50_fpn_1x
+```
+
+The model is trained on 4-node A100 machines with 2 images per gpu, which makes a batch size of 64 during training.
+
+### Testing commands
+
+In MMDetection's root directory, run the following command to test the model:
+
+```bash
+python tools/test.py projects/ViTDet/configs/mask-rcnn_vit-base_fpn_rpn-2conv_4conv1fc_ln-all_lsj-100e_coco.py ${CHECKPOINT_PATH}
+```
+
+## Results
+
+<!-- List the results as usually done in other model's README. [Example](https://github.com/open-mmlab/mmdetection/blob/3.x/configs/faster_rcnn/README.md#results-and-models)
+You should claim whether this is based on the pre-trained weights, which are converted from the official release; or it's a reproduced result obtained from retraining the model in this project. -->
+
+Here we provide the baseline version of ViTDet with ViT-base backbone.
+
+To find more variants, please visit the [official model zoo](https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet#pretrained-models).
+
+|                                             Method                                              |  Backbone   | Pretrained Model |  Training set  |   Test set   | #epoch | box AP |         Download         |
+| :---------------------------------------------------------------------------------------------: | :---------: | :--------------: | :------------: | :----------: | :----: | :----: | :----------------------: |
+| [Faster R-CNN dummy](configs/mask-rcnn_vit-base_fpn_rpn-2conv_4conv1fc_ln-all_lsj-100e_coco.py) | DummyResNet |        -         | COCO2017 Train | COCO2017 Val |   12   | 0.8853 | [model](<>) \| [log](<>) |
```

Comment:
update the performance when benchmark is done.
We should also make sure it is consistent with D2.